simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/ocppcommctrlr/NetworkConfigurationPriorityVariableAccessor.java

```diff
@@ -0,0 +1,126 @@
+package com.evbox.everon.ocpp.simulator.station.component.ocppcommctrlr;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.station.Station;
+import com.evbox.everon.ocpp.simulator.station.StationStore;
+import com.evbox.everon.ocpp.simulator.station.component.variable.SetVariableValidator;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableAccessor;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableGetter;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableSetter;
+import com.evbox.everon.ocpp.simulator.station.component.variable.attribute.AttributePath;
+import com.evbox.everon.ocpp.simulator.station.component.variable.attribute.AttributeType;
+import com.evbox.everon.ocpp.v20.message.centralserver.Component;
+import com.evbox.everon.ocpp.v20.message.centralserver.GetVariableResult;
+import com.evbox.everon.ocpp.v20.message.centralserver.SetVariableResult;
+import com.evbox.everon.ocpp.v20.message.centralserver.Variable;
+import com.evbox.everon.ocpp.v20.message.station.ReportDatum;
+import com.evbox.everon.ocpp.v20.message.station.VariableAttribute;
+import com.evbox.everon.ocpp.v20.message.station.VariableCharacteristics;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.evbox.everon.ocpp.v20.message.station.VariableCharacteristics.DataType.SEQUENCE_LIST;
+import static java.util.Collections.singletonList;
+import static org.apache.commons.lang3.StringUtils.isNumeric;
+
+public class NetworkConfigurationPriorityVariableAccessor extends VariableAccessor {
+
+    public static final String NAME = "NetworkConfigurationPriority";
+
+    NetworkConfigurationPriorityVariableAccessor(Station station, StationStore stationStore) {
+        super(station, stationStore);
+    }
+
+    private final Map<AttributeType, VariableGetter> variableGetters = ImmutableMap.<AttributeType, VariableGetter>builder()
```

Comment:
you can make it shorter `= ImmutableMap.of(AttributeType.ACTUAL, getActualValue())`

(applied for maps creation below as well)